etc/org.opencastproject.transcription.googlespeech.GoogleSpeechTranscriptionService.cfg

```diff
@@ -21,6 +21,17 @@ google.cloud.storage.bucket=
 # Filter out profanities from result. Default is false
 #google.speech.profanity.filter=false
 
+# Enable punctuations for transcription. Default is false
```

Comment:
These don't appear to be documented in the markdown docs.

Reply:
Thanks @gregorydlogan for taking a look. I have made the requested changes